### PR TITLE
Test/unit storage #113

### PR DESCRIPTION
**Pull request by Lussier115**

### CODE REVIEW

File: app/src/test/kotlin/StorageUnitTest.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        // Toggle the space remaining
+        mMainActivity?.config?.spaceRemainingOn = true
+        Assert.assertEquals(true, mMainActivity?.config?.spaceRemainingOn)
+
+        // Fake Storage Capacity
+        ShadowStatFs.registerStats("/tmp", 100, 1230, 1120)
+
+        mMainActivity?.setRecordingState(true)
+        Assert.assertEquals(true, mMainActivity?.findViewById<TextView>(R.id.space_remaining)?.isVisible())
+        Assert.assertEquals("4.38 Mb left", mMainActivity?.findViewById<TextView>(R.id.space_remaining)?.value)
```
Comment (anthonyle17) [inline app/src/test/kotlin/StorageUnitTest.kt:49]: This testFreeSpace is failing for me at line 47:
Expected: 4.38 Mb left
Actual: 0 byte left

**Review by anthonyle17 — CHANGES_REQUESTED** [review]
Check below comment for failing test

File: app/src/test/kotlin/StorageUnitTest.kt (hunk 1 above)
Comment (lkadian) [reply app/src/test/kotlin/StorageUnitTest.kt:49]: I get the same failing test. edit: I was wrong it was due to absolute path as Will said below

Comment (Lussier115) [reply app/src/test/kotlin/StorageUnitTest.kt:49]: The tests are working fine on my local machine? Does it tell you something like this: /tmp/robolectric-Method_firstTest6547737288131991820/external-cache ? whats the path of the test?

Comment (Lussier115) [reply app/src/test/kotlin/StorageUnitTest.kt:49]: I changed to use absolute path instead of using /tmp. Try to run it and tell me if it works for you.

Comment (lkadian) [reply app/src/test/kotlin/StorageUnitTest.kt:49]: Works now!

**Pull request merged**